# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libidl/bison-2.4.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Bison 2.4 update oncovered libIDL bugs:
http://bugzilla.gnome.org/show_bug.cgi?id=559395

  - Rene Rebe <rene@exactcode.de>

Index: parser.y
===================================================================
--- libIDL-0.8.11/parser.y.vanilla
+++ libIDL-0.8.11/parser.y
@@ -607,7 +607,7 @@
 			{ $<str>$ = "struct"; }
 			z_new_scope_catch '{'		{
 	g_hash_table_insert (__IDL_structunion_ht, $4, $4);
-	$$ = IDL_type_struct_new ($4, NULL);
+	$<tree>$ = IDL_type_struct_new ($4, NULL);
 }				member_list
 			'}' pop_scope			{
 	g_hash_table_remove (__IDL_structunion_ht, $4);
@@ -624,7 +624,7 @@
 				switch_type_spec
 			')' '{'				{
 	g_hash_table_insert (__IDL_structunion_ht, $4, $4);
-	$$ = IDL_type_union_new ($4, $7, NULL);
+	$<tree>$ = IDL_type_union_new ($4, $7, NULL);
 }				switch_body
 			'}' pop_scope			{
 	g_hash_table_remove (__IDL_structunion_ht, $4);
